source/javascripts/app/_property_key.js

```diff
@@ -0,0 +1,30 @@
+//= require ../lib/_jquery
+(function (global) {
+  'use strict';
+
+  function getQueryVariable(variable) {
+    var query = window.location.search.substring(1);
+    var vars = query.split('&');
+    for (var i = 0; i < vars.length; i++) {
+      var pair = vars[i].split('=');
+      if (decodeURIComponent(pair[0]) == variable) {
+        return decodeURIComponent(pair[1]);
+      }
+    }
+  }
+
+  function replaceExamplePropertyKey(key) {
+    $('span.s1').each(function() {
```

Comment:
I think `s1` is just a string type, but I might be wrong. Definitely something to keep an eye out for.